bin/start_aws.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -eu
+
+#
+# S3 bucket name is injected by Copilot as an environment variable
+# since it was created via copilot storage init --name pems-db, the variable is 'PEMSDB_NAME'
+S3_BUCKET_NAME="$PEMSDB_NAME"
+S3_FIXTURE_PATH="fixtures.json"
+LOCAL_FIXTURE_PATH="fixtures.json"
+
+echo "Downloading $S3_FIXTURE_PATH from bucket $S3_BUCKET_NAME"
+aws s3 cp "s3://${S3_BUCKET_NAME}/${S3_FIXTURE_PATH}" "${LOCAL_FIXTURE_PATH}"
+echo "Download complete"
```

Comment:
As a follow-up, let's look into _mounting_ the bucket as a volume that presents as a readable directory in the container. I assume this is possible with our setup (we do something similar in Azure).

This seems good for now :+1: